tests/e2e/support/objects/app-files/share/index.ts

```diff
@@ -67,7 +70,15 @@ export class Share {
   }
 
   async copyQuickLink(args: Omit<copyLinkArgs, 'page'>): Promise<void> {
-    await copyQuickLink({ ...args, page: this.#page })
+    this.#linksEnvironment = new LinksEnvironment()
+    const url = await copyQuickLink({ ...args, page: this.#page })
```

Comment:
```suggestion
    const url = await copyAndGetQuickLinkUrl({ ...args, page: this.#page })
```

can we name this function like this if possible? both have same function name i guess if i am not wrong.